core/src/apps/evolu/common.py

```diff
@@ -0,0 +1,35 @@
+def check_delegated_identity_proof(
+    proposed_value: bytes, header: bytes, arguments: list[bytes] | None = None
```

Comment:
```suggestion
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from buffer_types import AnyBytes


def check_delegated_identity_proof(
    proposed_value: AnyBytes, header: bytes, arguments: list[AnyBytes] | None = None
```

Use `AnyBytes` to prevent the need to parse to `bytes`

Reply:
I patched this in [`582f0e`](https://github.com/trezor/trezor-firmware/pull/5770/commits/582f0e494b7372ce6f96ac59cc6d6870a4ac7524)

What is the benefit of `AnyBytes` against just `bytes`?  Should I change more occurrences of `bytes` to `AnyBytes`?